protostar/commands/declare/declare_messages.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from protostar.io import StructuredMessage, LogColorProvider
+from protostar.starknet_gateway import SuccessfulDeclareResponse
+
+
+@dataclass
+class SuccessfulDeclareMessage(StructuredMessage):
```

Comment:
I think we should perhaps distinguish the two declare txns somehow in the console (V1 vs V2)

Reply:
Not sure if this is necessary: these are different version of the same transaction.

Reply:
#1724